README.md

```diff
@@ -61,12 +61,13 @@ services will not utilize that.
 
 Services should return a message with the UID and the validation message to the `uploadvalidation` topic:
 
-    {'hash': 'abcdef123456', 'validation': 'success'} # or 'validation': 'failure'
+    {'hash': 'abcdef123456', 'validation': 'success', 'validating_service': 'testareno'} # or 'validation': 'failure'
```

Comment:
@Glutexo To be fair, if we stand up an actual validator service (and it looks like we will be), then some of this will be pointless anyway. I'm going to let this PR sit out here for a bit until we officially nail down what the validator service is going to do.